browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -377,7 +377,7 @@ test.describe('applicant program index page', () => {
           await applicantQuestions.answerTextQuestion('second answer')
           await applicantQuestions.clickContinue()
           await applicantQuestions.submitFromReviewPage(true)
-          await applicantQuestions.returnToProgramsFromSubmissionPage()
+          await applicantQuestions.returnToProgramsFromSubmissionPage(true)
```

Comment:
**suggestion:** Can we add the /* northStarEnabled= */ comment here as well? (and not in scope but to the line above too :))